cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,55 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+//UpdateConfigHandler returns handler for configuration changes during runtime
+func UpdateConfigHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		_ = req.ParseForm()
+
+		setting := html.UnescapeString(req.Form.Get("setting"))
+		value := html.UnescapeString(req.Form.Get("value"))
+
+		log.Warnf("Got a request to change a setting: %s", setting)
```

Comment:
I'm not sure if this warrants a warning? Was this a requirement and how other agents do it? Otherwise `Info` is better I think. Warnings are when something goes wrong or is about to.

Lastly, I think there is too much logging in this handler.

Reply:
I'll remove this and leave only `"Switched to..."` one